src/fold/transformations/features.py

```diff
@@ -0,0 +1,43 @@
+from typing import List, Union
+
+import holidays
+import pandas as pd
+
+from ..utils.list import wrap_in_list
+from .base import Transformation, fit_noop
+
+
+class AddHolidayFeatures(Transformation):
+    """
+    Adds holiday features for given regions as a column.
```

Comment:
could you explain what's the format exactly?

Reply:
is it one-hot? does it also mark what kind of holiday is it, or just that it is a holiday? can we add option to do either?

Reply:
An integer for the type of holiday?

Reply:
yes! that'd be great! not all holidays may be equal, and if someone has decades of data, it could be a useful distinction.